### ISSUE TITLE
Buttons on response body area

### ISSUE THREAD

**Issue by PatrykKuniczak** [swagger-api/swagger-ui#8572]
### Q&A (please complete the following information)
 - OS: Windows 11
 - Browser: Google Chrome
 - Version: 112.0.5615.86
 - Method of installation: npm
 - Swagger-UI version: "@nestjs/swagger": "6.2.1",
 - Swagger/OpenAPI version: OpenAPI 3.0

### Content & configuration

Example Swagger/OpenAPI definition:
Unnecesary, run default.

Swagger-UI configuration options:
Unnecesary, run default.

### Describe the bug you're encountering
I have a problem with layout, on response body the buttons is displayed on text from response.

### To reproduce...

Run default project with swagger.

### Expected behavior
Buttons container should be displaying next to response body conatainer.

### Screenshots
![image](https://user-images.githubusercontent.com/64608510/232300830-440733aa-b060-456e-a517-54ba58c58968.png)

![image](https://user-images.githubusercontent.com/64608510/232300848-159488f5-aa6a-455f-a7ca-e06e16d275d1.png)

**Comment by PatrykKuniczak** [comment]
@Tobbiloba "Let's get it start it aha, let's get it start it again" :)

**Comment by hkosova** [comment]
This looks like a duplicate of #7541.

**Comment by PatrykKuniczak** [comment]
@hkosova Yeah, it's duplicated, but why this issue still be unresolved, it's seems easy to fix.
